### DISCUSSION TITLE
msubs of sympy.physics.mechanics

### DISCUSSION THREAD

**Discussion by Peter230655** [sympy/sympy#26725] [Q&A]
msubs allows multiple dictionaries, say expr1 = me.msubs(expr, dict_1, ... , dict_n)
I which sequence are the dictionaries applied to expr?
This may be important if dict_i has items, which should be substituted by dict_j.
I played around, but frankly did not manage to find an answer, also I could not find an answer in the docs.
**Thanks for any clarification!**

**Comment by moorepants** [comment]
The order is in the same order as you provide the dictionaries but they are all merged into one dictionary with `sympy.utilities.dict_merge`.

**Comment by Peter230655** [comment]
**Thanks!** Now clear why what I tried initially did not work.
